cmd/dp-search-api/main.go

```diff
@@ -60,10 +62,26 @@ func main() {
 		}
 	}
 
+	elasticHTTPClient := dphttp.NewClient()
 	elasticSearchClient := elasticsearch.New(cfg.ElasticSearchAPIURL, dphttp.NewClient(), cfg.SignElasticsearchRequests, esSigner, cfg.AwsRegion, cfg.AwsService)
 	transformer := transformer.New()
+	svcList := service.NewServiceList(&service.Init{})
+
+	// Get HealthCheck
+	ctx := context.Background()
+
+	if err != nil {
+		log.Event(nil, "could not instantiate healthcheck", log.FATAL, log.Error(err))
+		os.Exit(1)
+	}
```

Comment:
I think this can be deleted now as its no longer needed.

Reply:
Oops. Sorry about that.